src/components/OmouComponents/IdleLogout.js

```diff
@@ -117,6 +131,7 @@ const IdleLogout = () => {
         })
     };
     
+    // Refreshes the token every 15 minutes
     const refreshTokenAfter15Minutes = () => {
```

Comment:
The comment here isn't very helpful - it's just what you're saying in the function name. Why are you refreshing every 15 minutes?